jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc

```diff
@@ -0,0 +1,102 @@
+= WFLY-14559 Add six new RESTEasy context parameters
+:author:            Ron Sigal
+:email:             rsigal@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Since the RESTEasy context parameters were first exposed in the wildfly/jaxrs management model
+(https://github.com/wildfly/wildfly-proposals/blob/main/jaxrs/WFLY-12298_Change_RESTEASY_settings.adoc),
+several new parameters have been introduced, of which the following should be exposed to the user:
+
+ * resteasy.match.cache.enabled:                       [RESTEASY-2646]
+ * resteasy.match.cache.size:                          [RESTEASY-2646]
+ * resteasy.original.webapplicationexception.behavior: [RESTEASY-2782]
+ * resteasy.patch.filter.disabled:                     [RESTEASY-2280]
+ * resteasy.patch.filter.legacy:                       [RESTEASY-2849]
+ * resteasy.proxy.implement.all.interfaces:            [RESTEASY-2866]
+
+The wildfly/jaxrs module should be extended to expose these new parameters.
+
+[Note. RESTEasy uses '.' as a separator in parameter names. WildFly/jaxrs uses '-'. 
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/EAP7-1655[EAP7-1655]
+* {issue-base-url}/WFLY-14559[WFLY-14559]
+
+=== Related Issues:
+
+* {issue-base-url}/RESTEASY-2280[RESTEASY-2280]
+* {issue-base-url}/RESTEASY-2646[RESTEASY-2646]
+* {issue-base-url}/RESTEASY-2782[RESTEASY-2782]
+* {issue-base-url}/RESTEASY-2849[RESTEASY-2849]
+* {issue-base-url}/RESTEASY-2866[RESTEASY-2866]
```

Comment:
Minor nit, this should just be `https://issues.redhat.com/browse`. We migrated away from the `issue-base-url` for ease of copy/paste when reviewing.